pyproject.toml

```diff
@@ -18,7 +18,7 @@ classifiers = [
 dependencies = [
   "pymatgen>=2024.10.22",
   "scikit-image>=0.19.3",
-  "numpy<2",
+  "numpy>=2.0.0,<3.0.0",
```

Comment:
note; CIs passed without any code modification, therefore it would be possible to allow users to use both NumPy v1 and v2.